llvm/test/CodeGen/AIE/aie2/internalize.ll

```diff
@@ -0,0 +1,123 @@
+; RUN: opt -O1 -S -mtriple=aie2-none-unknown-elf -aie2-internalize-symbols < %s | FileCheck --check-prefix=ONLYMAIN %s
+; RUN: opt -O1 -S -mtriple=aie2-none-unknown-elf -aie2-internalize-symbols -aie2-internalize-skip-functions="skipfunc1" -aie2-internalize-skip-functions="skipfunc2" -aie2-internalize-skip-functions="skipfunc3" < %s | FileCheck --check-prefix=FUNCLIST %s
```

Comment:
Maybe use `cl::CommaSeparated` to keep the command line shorter?
i.e. allow to pass the symbols as
```
-aie2-internalize-skip-functions="skipfunc1,skipfunc2,skipfunc3"
```